Changelog.md

```diff
@@ -7,6 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 
+## [0.38.0] - 2024-03-31
+
+With over two years of collecing breaking changes (since the `0.37.0` release in March 2022), March 2024 marks the next breaking release of `ash`.  This release introduces an overhaul of all Vulkan structures, restructures modules around extensions, and separates extension wrappers between `Instance` and `Device` functions.  The crate contains all bindings defined by the latest `1.3.281` Vulkan specification, and many old and new extensions have received a hand-written extension wrapper.  For a full overview of all individual changes, see the list at the end of this post.
```

Comment:
```suggestion
With over two years of collecting breaking changes (since the `0.37.0` release in March 2022), March 2024 marks the next breaking release of `ash`.  This release introduces an overhaul of all Vulkan structures, restructures modules around extensions, and separates extension wrappers between `Instance` and `Device` functions.  The crate contains all bindings defined by the latest `1.3.281` Vulkan specification, and many old and new extensions have received a hand-written extension wrapper.  For a full overview of all individual changes, see the list at the end of this post.
```
small typo

Reply:
Thanks - yeah I'm subscribed to the GitHub issue to add spell-checking support to `helix` - do tend to make these from time to time and not see them without squiglies :)